tls/s2n_config.c

```diff
@@ -1071,3 +1072,19 @@ int s2n_config_set_recv_multi_record(struct s2n_config *config, bool enabled)
 
     return S2N_SUCCESS;
 }
+
+/* Indicates if the connection should attempt to enable kTLS. */
+int s2n_config_ktls_enable(struct s2n_config *config, s2n_ktls_mode ktls_mode)
+{
+    POSIX_ENSURE_REF(config);
+
+    config->ktls_mode_requested = ktls_mode;
+    return S2N_SUCCESS;
+}
+
+S2N_RESULT s2n_config_is_ktls_requested(struct s2n_config *config, s2n_ktls_mode ktls_mode, bool *requested)
```

Comment:
I understand why you would need to check the connection to see if KTLS has actually been enabled, but why would someone need to query the config to determine if they've requested it? Or is this not a public API?